bower.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "temp",
+  "version": "1.0.0",
+  "dependencies": {
+    "threejs": "r55",
```

Comment:
This... I couldn't figure out why you were not using r71 (latest) Then when I plugged it in nothing worked. So the author pulled the canvas render function out of the core lib.. weird. I set it to r55 for now.